applications/GeoMechanicsApplication/tests/cpp_tests/custom_constitutive/test_mohr_coulomb_with_tension_cutoff.cpp

```diff
@@ -8,29 +8,48 @@
 //  License:         geo_mechanics_application/license.txt
 //
 //  Main authors:    Gennady Markelov
+//                   Mohamed Nabi
 //
 
-#include "custom_constitutive/coulomb_yield_surface.h"
 #include "custom_constitutive/mohr_coulomb_with_tension_cutoff.h"
 #include "custom_constitutive/plane_strain.h"
-#include "custom_constitutive/three_dimensional.h"
-#include "custom_utilities/stress_strain_utilities.h"
 #include "geo_mechanics_application_variables.h"
 
 #include "tests/cpp_tests/geo_mechanics_fast_suite.h"
 #include "tests/cpp_tests/test_utilities.h"
 
-#include "utilities/math_utils.h"
-
 #include <boost/numeric/ublas/assignment.hpp>
-#include <sstream>
-#include <string>
 
 using namespace Kratos;
-using namespace std::string_literals;
 
 namespace Kratos::Testing
 {
+Properties CreateProperties(double FrictionAngle, double Cohesion, double DilatancyAngle, double TensileStrength)
+{
+    Properties properties;
+    properties.SetValue(GEO_FRICTION_ANGLE, FrictionAngle);
+    properties.SetValue(GEO_COHESION, Cohesion);
+    properties.SetValue(GEO_DILATANCY_ANGLE, DilatancyAngle);
+    properties.SetValue(GEO_TENSILE_STRENGTH, TensileStrength);
+    return properties;
+}
+
+Vector CalculateMappedStressVector(Vector&                       cauchy_stress_vector,
+                                   ConstitutiveLaw::Parameters&  parameters,
+                                   MohrCoulombWithTensionCutOff& law)
+{
+    Vector strain_vector = ZeroVector(4);
+    parameters.SetStrainVector(strain_vector);
+    parameters.SetStressVector(cauchy_stress_vector);
+    ProcessInfo process;
```

Comment:
We can make this object `const`:
```suggestion
    const auto dummy_process_info = ProcessInfo{};
```

Reply:
done